src/redist/targets/BundledManifests.targets

```diff
@@ -3,12 +3,12 @@
     <NetFeatureBand>8.0.100</NetFeatureBand>
   </PropertyGroup>
   <ItemGroup>
-    <BundledManifests Include="Microsoft.NET.Sdk.Android" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinAndroidWorkloadManifestVersion)" />
-    <BundledManifests Include="Microsoft.NET.Sdk.iOS" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinIOSWorkloadManifestVersion)" />
-    <BundledManifests Include="Microsoft.NET.Sdk.MacCatalyst" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinMacCatalystWorkloadManifestVersion)" />
-    <BundledManifests Include="Microsoft.NET.Sdk.macOS" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinMacOSWorkloadManifestVersion)" />
-    <BundledManifests Include="Microsoft.NET.Sdk.Maui" FeatureBand="$(MauiFeatureBand)" Version="$(MauiWorkloadManifestVersion)" />
-    <BundledManifests Include="Microsoft.NET.Sdk.tvOS" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinTvOSWorkloadManifestVersion)" />
+    <BundledManifests Include="Microsoft.NET.Sdk.Android" FeatureBand="$(MauiFeatureBand)" Version="$(XamarinAndroidWorkloadManifestVersion)" Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
Does this check need to be source-build specific or can it be tied to non-windows platforms?

Reply:
It needs to be source build specific as MAUI / mobile is supported on non windows platforms.

Reply:
We ship all the manifests on all platforms, even if they aren't theoretically very useful there (e.g. I have a microsoft.net.sdk.maccatalyst folder in `c:\program files\dotnet\sdk-manifests\`)

As I understand it, this change should allow us to use source-built manifests for the two repos where we have them (`emsdk` and `runtime`)

Reply:
If there is a use case for a workload on linux, then it should be included in source-build.  if a workload is not intended to be used on specific platforms, it should be conditional base on the platform rather than source-build.  The desire is to not introduce more differences in source-build rather we want to be unifying source-build and the Microsoft build.

It would be acceptable to add source-build conditions for preview 1 as long as we have issues logged to resolve the differences within the 8.0 timeframe.

cc @mmitche

Reply:
Yes, workloads are used on Linux

Reply:
> Yes, workloads are used on Linux

Yes, understood, I am not questioning the usefulness of workloads on linux.  I am questioning if **all** of them are useful.  If some are not, we should condition based on platform not source-build.  If we think they all apply, then the manifests should be produced as part of source-build rather conditioning them out of source-build.

Reply:
The ones I disabled are the ones we can't source-build right now (they come from Maui). When we're in a position to source-build Maui (even if that means for a manifest-only build, the way emsdk is right now) then I'd expect us to ingest all of them on all platforms, matching the behaviour of official proprietary binary releases which ship all the manifests on all platforms

Reply:
Yes, all of them are useful on linux. The challenge is that MAUI and the Android/iOS manifests are built in repos not currently in source build.  It's unclear how much work it would take to resolve in the .NET 8 timeframe.

@MichaelSimons, should we take this PR or roll with what we have currently?

Reply:
> should we take this PR or roll with what we have currently?

We should take this PR to unblock preview 1 and then open an issue to source-build the Maui workloads and remove these conditions.